include/world_builder/topography/uniform.h

```diff
@@ -0,0 +1,87 @@
+/*
+  Copyright (C) 2018-2025 by the authors of the World Builder code.
+
+  This file is part of the World Builder.
+
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU Lesser General Public License as published
+   by the Free Software Foundation, either version 2 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU Lesser General Public License for more details.
+
+   You should have received a copy of the GNU Lesser General Public License
+   along with this program.  If not, see <https://www.gnu.org/licenses/>.
+*/
+
+#ifndef WORLD_BUILDER_TOPOGRAPHY_MINIMUM_DEPTH_H
+#define WORLD_BUILDER_TOPOGRAPHY_MINIMUM_DEPTH_H
+
+#include "world_builder/topography/interface.h"
+
+#include "world_builder/utilities.h"
+#include "world_builder/objects/natural_coordinate.h"
+
+
+namespace WorldBuilder
+{
+
+  namespace Topography
+  {
+    /**
+     * This implements a minimum depth topography. The minimum depth topography
+     * finds the minimum depth of a feature and uses that for the topography.
+     */
+    class Uniform final : public Interface
+    {
+      public:
+        /**
+         * Constructor
+         */
+        Uniform(WorldBuilder::World *world);
+
+        /**
+         * Destructor
+         */
+        ~ Uniform() override final;
+
+        /**
+         * declare and read in the world builder file into the topography class
+         */
+        static
+        void declare_entries(Parameters &prm, const std::string &parent_name = "");
+
+        /**
+         * declare and read in the world builder file into the topography class
+         */
+        void parse_entries(Parameters &prm) override final;
+
+        /**
+         * declare and read in the world builder file into the topography class
```

Comment:
These comments are all the same, which makes them less informative. Maybe make them more specific to the functions?